src/adapter/src/catalog/storage.rs

```diff
@@ -227,9 +233,16 @@ async fn migrate<S: Append>(
                 ComputeInstanceReplicaValue {
                     compute_instance_id: DEFAULT_COMPUTE_INSTANCE_ID,
                     name: "default_replica".into(),
-                    config: json!({"Managed": {
-                        "size": bootstrap_args.default_cluster_replica_size,
-                    }})
+                    config: json!({
+                        "persisted_logs": "Default",
+                        "location": {
+                            "Managed": {
+                                "size": bootstrap_args.default_cluster_replica_size,
+                                "availability_zone": "",
```

Comment:
I may be misunderstanding the old migration, but should this be `bootstrap_args.default_availability_zone.clone()`

from
```
                    managed.insert(
                        AZ_KEY.to_string(),
                        serde_json::Value::String(bootstrap_args.default_availability_zone.clone()),
                    );
```

Reply:
Ah yes! Good catch.